src/chart.zoom.js

```diff
@@ -527,10 +527,16 @@ var zoomPlugin = {
 			var endX = Math.max(beginPoint.clientX, endPoint.clientX) - offsetX;
 			var rectWidth = endX - startX;
 
+			var dragOptions = chartInstance.options.zoom.drag;
 
-			ctx.fillStyle = 'rgba(225,225,225,0.3)';
-			ctx.lineWidth = 5;
+			ctx.fillStyle = dragOptions.backgroundColor || 'rgba(225,225,225,0.3)';
```

Comment:
This line seems like it could fail if someone sets `drag: true`. It seems potentially problematic that `drag` can contain either a boolean or an object

Reply:
Mmh if `drag: true`, then `dragOptions.backgroundColor` will be `undefined` and the default color will be used, so I don't think it's an issue. Nevertheless, for better readability and maintainability, we could use another property as you suggest. `dragOptions` for instance?

```js
{
  zoom: {
    dragOptions: {
      borderColor: 'rgba(225,225,225,0.3)'
      borderWidth: 5,
      backgroundColor: 'rgb(225,225,225)'
    }
  }
}
```

Reply:
Ah. You're right. The code will work as written. I'm fine with leaving it as is

If we did want to change it another option could be to do something like the below, but I think it's probably fine the way you have it
```
{
  zoom: {
    drag: {
      enabled: true
      borderColor: 'rgba(225,225,225,0.3)'
      borderWidth: 5,
      backgroundColor: 'rgb(225,225,225)'
    }
  }
}
```

Reply:
Ok so I leave as its is. Checking `drag.enabled` could introduce breaking changes in some configurations (if any user set anything other than a boolean as `drag`), it may be handled in another MR.